lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_ceremony_country.govspeak.erb

```diff
@@ -295,17 +272,15 @@
     $D
 
 
+  <% elsif calculator.ceremony_country == 'bulgaria' %>
```

Comment:
I'd change this statement to `elsif calculator.ceremony_country != 'bulgaria'`. Then you change remove the `else` clause directly below. This should make the statement read better. At first glance, I thought you'd forgotten to add some code!